src/libs/actions/IOU.ts

```diff
@@ -3383,6 +3390,11 @@ function submitReport(expenseReport: OnyxTypes.Report) {
                 statusNum: CONST.REPORT.STATUS_NUM.SUBMITTED,
             },
         },
+        {
+            onyxMethod: Onyx.METHOD.SET,
```

Comment:
I dont think we need to use SET in this case, do we?
```suggestion
            onyxMethod: Onyx.METHOD.MERGE,
```

Reply:
Done - 655d9de.